config/test-1.yaml

```diff
@@ -1,4 +1,5 @@
 listen:
+  hostname: ''
```

Comment:
We don't need this parameter in test configurations

Reply:
Are you sure @Chocobozzz  ? I think that would fail to start in that case.

Reply:
Test configurations inherit the `default` configuration, so if you do https://github.com/Chocobozzz/PeerTube/pull/514#discussion_r181984904 it should work :)